dg/utils/ssh.py

```diff
@@ -71,7 +73,7 @@ async def execute(self, command: str, print_output: bool = True) -> str:
         """
 
         if self._connection is None:
-            raise Exception("Not connected to " + self._hostname)
+            raise DataGateCLIException("Not connected to " + self._hostname)
```

Comment:
Should we add `.` after all exception messages? I won't spam the PR by adding suggestions for this everywhere, either way is okay in my opinion. If so:
```suggestion
            raise DataGateCLIException(f"Not connected to {self._hostname}.")
```

Reply:
Here you brought up the question I had above yourself. I am reluctant to add a comma at the end of incomplete sentences. However, I would be fine with rules like the following ones:
- Error messages must be complete sentences with a full stop.
- Error messages must be complete sentences with a full stop or incomplete sentences without a full stop.
What do you think?

Reply:
@jensmueller-com Agreeing to
```Error messages must be complete sentences with a full stop or incomplete sentences without a full stop.```

So, adding a `.` where applicable.